test/unit/behaviours/tenancy-start-west-midlands-check.test.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+const reqres = require('reqres');
+const Behaviour = require('../../../apps/right-to-rent-check/behaviours/tenancy-start-west-midlands-check');
+
+describe('apps/behaviours/tenancy-start-west-midlands-check', () => {
+  it('exports a function', () => {
+    expect(Behaviour).to.be.a('function');
+  });
+
+  describe('getNextStep()', () => {
+    class Base {
+      getNextStep() {}
+    }
+    let req;
+    let res;
+    let sessionModel;
+    let instance;
+    let ValidTenants;
+
+    beforeEach(() => {
+      sessionModel = {
+        get: sinon.stub()
+      };
+      res = reqres.res();
+      req = reqres.req({sessionModel});
+      sinon.stub(Base.prototype, 'getNextStep');
+      ValidTenants = Behaviour(Base);
+      instance = new ValidTenants();
+    });
+
+    afterEach(() => {
+      Base.prototype.getNextStep.restore();
+    });
+
+    it('calls super getNextStep if valid-tenancy is true', () => {
```

Comment:
when you write test descriptions try to keep them human readable. E.g. `calls the parent method when the tenancy is valid`